backend/coreapp/views/project.py

```diff
@@ -239,6 +302,50 @@ def pr(self, request: Request, pk: str) -> Response:
         )
         return Response({"url": response.html_url})
 
+    @action(detail=True, methods=["GET", "PUT"])
+    def members(self, request: Request, pk: str) -> Response:
```

Comment:
So the client has to send a full list of project members each time a single person is to be added? I'm not sure I like that model. Can't we just have one command to add members and another to remove them? or even keep the single PUT but have two parameters, one for people to add and one for people to remove?

Reply:
Bear in mind this "full list" is just usernames. 

> Can't we just have one command to add members and another to remove them?

We could implement this as a ViewSet, I guess?
`POST /members` - add member
`DELETE /members/username` - remove member
`PATCH /members/username` - change member's role (later)

> or even keep the single PUT but have two parameters, one for people to add and one for people to remove?

Violates PUT semantics